packages/patternfly-4/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -257,10 +276,13 @@ class Tabs extends React.Component<TabsProps & InjectedOuiaProps, TabsState> {
             <AngleRightIcon />
           </button>
         </Component>
-        {React.Children.map(children, (child: any, index) =>
-          !child.props.children ? null : (
-            <TabContent key={index} activeKey={activeKey} child={child} id={child.props.id || uniqueId} />
-          )
+        {React.Children.map(children, (child: any, index) => {
+          if (!child.props.children || (unmountOnExit && child.props.eventKey !== activeKey) || (mountOnEnter && shownKeys.indexOf(child.props.eventKey) === -1)) {
+            return undefined;
```

Comment:
When you don't want to render a component, you typically return null. In this case; however, perhaps you can return only when there is tab content? That is, skip returning null or undefined.

Reply:
Yeah, @dlabaj asked me to do this. Do you have a preference, Don? Thanks.

Reply:
I don't think you need to return anything, but the items you intend to render here. Easy enough to try and see how it works...

Reply:
@rebeccaalpert Since react docs say to return null we should do that instead.  In other frameworks I was use to it being undefined.

Reply:
All set.